### PR TITLE
feat: Add energy results table to Unity Catalog

### PR DESCRIPTION
**Pull request by BjarkeMeier**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are executed locally
- [x] Subsystem tests have been tested (by manually deploying to `dev_002`) (NOTE: the table wasn't created in UC)

### CODE REVIEW

**Review by JesperGomes — DISMISSED** [review]

File: source/databricks/calculation_engine/package/datamigration_hive/current_state_scripts/views/hive_create__calculation_results__energy_result_per_neighbor_ga_v1.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CREATE VIEW {CALCULATION_RESULTS_DATABASE_NAME}.exchange_per_neighbor_ga_v1 as
+CREATE VIEW IF NOT EXISTS {CALCULATION_RESULTS_DATABASE_NAME}.exchange_per_neighbor_ga_v1 as
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/datamigration_hive/current_state_scripts/views/hive_create__calculation_results__energy_result_per_neighbor_ga_v1.sql:1]: why is this needed?

**Review by JesperGomes — APPROVED** [review]

File: source/databricks/calculation_engine/package/datamigration_hive/current_state_scripts/views/hive_create__calculation_results__energy_result_per_neighbor_ga_v1.sql (hunk 1 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration_hive/current_state_scripts/views/hive_create__calculation_results__energy_result_per_neighbor_ga_v1.sql:1]: Fra Morten: Eksempelvis hvis en tabel er blevet fjernet manuelt, så kan den genoprettes med current state

Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration_hive/current_state_scripts/views/hive_create__calculation_results__energy_result_per_neighbor_ga_v1.sql:1]: And I had some test problems of which I was unsure whether they were related to this or not.

**Review by Renetnielsen — APPROVED** [review]

File: source/databricks/calculation_engine/package/datamigration_hive/current_state_scripts/views/hive_create__calculation_results__energy_result_per_neighbor_ga_v1.sql (hunk 1 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration_hive/current_state_scripts/views/hive_create__calculation_results__energy_result_per_neighbor_ga_v1.sql:1]: We can add a task to the story to investigate if you want?
